### ISSUE TITLE
CalendarPanel date cells's aria-label should announce "today" if the date is today's date

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#11582]
[Forum post](https://forum.bryntum.com/viewtopic.php?t=33434)

Hi there,

When navigating the Bryntum Calendar's date picker using a screen reader, the "Today" date is not announced when focused. This information is available for the normal users as the today's date is displayed in a yellow ring, but missing for the Assistive Technology users.

![Bryntum_today_date_issue.png](https://forum.bryntum.com/download/file.php?id=35426)
![Bryntum_today_date_issue.png](https://forum.bryntum.com/download/file.php?id=35426)
